kubernetes/kserve/README.md

```diff
@@ -1,55 +1,109 @@
-## End to End Documentation for Torchserve - KServe Model Serving
+# End to End Documentation for Torchserve - KServe Model Serving
 
 The documentation covers the steps to run Torchserve inside the KServe environment for the mnist model.
 
 Currently, KServe supports the Inference API for all the existing models but text to speech synthesizer and it's explain API works for the eager models of MNIST,BERT and text classification only.
 
-### Docker Image Dev Build
-
-```
-DOCKER_BUILDKIT=1 docker build -f Dockerfile.dev -t pytorch/torchserve-kfs:latest-dev .
-```
-
-### Docker Image Building
+## Docker Image Building
 
 - To create a CPU based image
 
-```
+```bash
 ./build_image.sh
 ```
 
 - To create a CPU based image with custom tag
 
-```
+```bash
 ./build_image.sh -t <repository>/<image>:<tag>
 ```
 
 - To create a GPU based image
 
-```
+```bash
 ./build_image.sh -g
 ```
 
 - To create a GPU based image with custom tag
 
-```
+```bash
 ./build_image.sh -g -t <repository>/<image>:<tag>
 ```
 
-### Running Torchserve inference service in KServe cluster
+### Docker Image Dev Build
+
+```bash
+DOCKER_BUILDKIT=1 docker build -f Dockerfile.dev -t pytorch/torchserve-kfs:latest-dev .
+```
+
+## Running Torchserve inference service in KServe cluster
+### Create Kubernetes cluster with eksctl
```

Comment:
Suggestion to give  a link to eksctl

Reply:
a) added
b) this steps creates the k8s cluster.